dbms/src/Storages/StorageMySQL.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <Storages/IStorage.h>
+#include <sparsehash/dense_hash_map>
+
+#include <Poco/Net/SocketAddress.h>
+
+#include <mysqlxx/Pool.h>
+
+namespace DB
+{
+
+inline std::string splitHostPort(const char * host_port, int & port)
```

Comment:
Function is used only in StorageMySQL. Probably, it's better to move it to `StorageMySQL.cpp` and make `static` ?

Reply:
Found in `TableFunctionMySQL.cpp`. My bad.